engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -164,4 +169,39 @@ class ShadowWorkflowActor(workflowId: WorkflowId, startMode: StartMode, wdlSourc
       log.warning(s"$tag received an unhandled message: $unhandledMessage")
       stay
   }
+
+  private def assignCallsToBackends(calls: Seq[Call], workflowOptions: WorkflowOptions): Map[Call, String] = {
+    val backendKey = "backend"
+    // ------------------------------------------------------------------//
+    // TODO: This should be obtained from the backend config factory. Part of issue #649
+    // This is currently a stub. Should be removed as part of #649 (?)
+    val listOfPluggedInBackends = Seq("local", "sge", "jes", "htcondor")
```

Comment:
I prefer to not directly instantiate Seqs. One can directly instantiate a List and have the exact same result and lose some fuzziness